docs/core/porting/upgrade-assistant-wpf-framework.md

```diff
@@ -11,83 +11,434 @@ The [.NET Upgrade Assistant](upgrade-assistant-overview.md) is a command-line to
+[16:55:21 WRN] No version of System.Configuration.ConfigurationManager found that supports ["net452"]; leaving unchanged
+[16:55:21 INF] Upgrade step Clean up NuGet package references applied successfully
+Please press enter to continue...
+```
+
+#### Handle the TFM
+
+The tool next changes the [TFM](../../standard/frameworks.md) from .NET Framework to the suggested SDK. In this example, it's `net6.0-windows`.
+
+```
+[16:56:36 INF] Applying upgrade step Update TFM
+[16:56:36 INF] Recommending TFM net6.0-windows because of dependency on project C:\code\migration\wpf\sampleApp\StarVoteControl\StarVoteControl.csproj
+[16:56:40 INF] Updated TFM to net6.0-windows
+[16:56:40 INF] Upgrade step Update TFM applied successfully
+Please press enter to continue...
+```
+
+#### Upgrade NuGet packages
 
 Next, the tool updates the project's NuGet packages.
 
-:::image type="content" source="media/upgrade-assistant-wpf-framework/update-nuget-packages.png" alt-text=".NET Upgrade Assistant update NuGet packages":::
+```
+[16:58:06 INF] Applying upgrade step Update NuGet Packages
+[16:58:06 INF] Removing outdated assembly reference: System.Web
+[16:58:06 INF] Removing outdated assembly reference: System.Web.Extensions
+[16:58:06 INF] Removing outdated package reference: Microsoft.CSharp, Version=4.7.0
+[16:58:06 INF] Removing outdated package reference: System.Data.DataSetExtensions, Version=4.5.0
+[16:58:06 INF] Removing outdated package reference: EntityFramework, Version=6.2.0
+[16:58:06 INF] Adding package reference: EntityFramework, Version=6.4.4
+[16:58:06 INF] Adding framework reference: Microsoft.AspNetCore.App
+[16:58:11 INF] Upgrade step Update NuGet Packages applied successfully
+Please press enter to continue...
+```
 
-Once the packages are updated, the next step is to add template files, if any. In this case, there are no template files that need to be added. This step continues and migrates app config files and updates C# source to apply fixes, as shown below. This project didn't need any config file or source code changes, so these steps proceeded automatically.
+#### Templates, config, and code files
 
-:::image type="content" source="media/upgrade-assistant-wpf-framework/add-template-files.png" alt-text=".NET Upgrade Assistant add template files and migrate config":::
+The next few steps may be skipped automatically by the tool if the tool determines there isn't anything to do for your project.
 
-Since this is the last project, the next step, "Move to next project", prompts to complete the process of migrating the entire solution.
+Once the packages are updated, the next step is to update any template files. In this example, there are no template files that need to be updated or added to the project. This step is skipped and the next step is automatically started: Upgrade app config files. In this example, the step only needs to convert the connection strings:
```

Comment:
Template files only apply to web apps, currently (program.cs, for example, for ASP.NET Core apps), which is why there's nothing to do in this case.